package.json

```diff
@@ -1,29 +1,113 @@
   "license": "Apache 2.0 License + Common Clause Condition 1.0",
-  "devDependencies": {
-    "mocha": "^8.1.3"
+  "bugs": {
+    "url": "https://github.com/vesoft-inc/nebula-node/issues"
   },
+  "homepage": "https://github.com/vesoft-inc/nebula-node#readme",
   "dependencies": {
-    "thrift": "^0.9.3",
-    "browser-or-node": "^1.3.0",
-    "chai": "^4.2.0",
+    "bindings": "^1.5.0",
+    "browser-or-node": "^1.2.1",
     "isomorphic-ws": "^4.0.1",
-    "q": "^1.5.1",
-    "ws": "^7.3.1"
+    "node-int64": "^0.4.0",
+    "q": "^1.5.0",
+    "ws": "^5.2.2",
+    "lodash": "^4.17.21",
+    "moment": "^2.29.1",
+    "uuid": "^8.3.2"
+  },
+  "keywords": [
+    "nebula",
+    "nodejs sdk"
+  ],
+  "engines": {
+    "node": ">=10.0.0"
+  },
+  "devDependencies": {
+    "@types/benchmark": "^2.1.1",
+    "@types/thrift": "^0.10.11",
+    "benchmark": "^2.1.4",
+    "ts-node": "^10.0.0",
+    "@babel/cli": "^7.14.5",
+    "@babel/core": "^7.14.6",
+    "@babel/node": "^7.14.5",
+    "@babel/plugin-proposal-decorators": "^7.14.5",
+    "@babel/plugin-proposal-do-expressions": "^7.14.5",
+    "@babel/plugin-proposal-export-default-from": "^7.14.5",
+    "@babel/plugin-proposal-function-bind": "^7.14.5",
+    "@babel/plugin-proposal-function-sent": "^7.14.5",
+    "@babel/plugin-proposal-pipeline-operator": "^7.14.5",
+    "@babel/plugin-proposal-throw-expressions": "^7.14.5",
+    "@babel/plugin-syntax-import-meta": "^7.10.4",
+    "@babel/plugin-transform-runtime": "^7.14.5",
+    "@babel/preset-env": "^7.14.5",
+    "@babel/preset-typescript": "^7.14.5",
+    "@istanbuljs/nyc-config-babel": "^3.0.0",
+    "@istanbuljs/nyc-config-typescript": "^1.0.1",
+    "@types/bluebird": "^3.5.35",
+    "@types/chai": "^4.2.18",
+    "@types/esm": "^3.2.0",
+    "@types/glob": "^7.1.3",
+    "@types/inquirer": "^7.3.1",
+    "@types/lodash": "^4.14.170",
+    "@types/mocha": "^8.2.2",
+    "@types/node": "^14.17.3",
+    "@types/sinon": "^9.0.11",
+    "@types/sinon-chai": "^3.2.5",
+    "@typescript-eslint/eslint-plugin": "^4.27.0",
+    "@typescript-eslint/parser": "^4.27.0",
+    "assert": "^2.0.0",
+    "babel-eslint": "^10.1.0",
+    "babel-plugin-add-module-exports": "^1.0.4",
+    "babel-plugin-istanbul": "^6.0.0",
+    "babel-plugin-transform-typescript-metadata": "^0.3.2",
+    "bluebird": "^3.7.2",
+    "chai": "^4.3.4",
+    "chalk": "^4.1.1",
+    "core-js": "^2.6.10",
+    "cowsay": "^1.5.0",
+    "cross-env": "^7.0.3",
+    "eslint": "^7.28.0",
+    "eslint-config-airbnb-base": "^14.2.0",
+    "eslint-config-semistandard": "^15.0.1",
+    "eslint-config-standard": "^16.0.3",
+    "eslint-plugin-import": "^2.23.4",
+    "eslint-plugin-jsdoc": "^32.3.4",
+    "eslint-plugin-node": "^11.0.0",
+    "eslint-plugin-prefer-arrow": "^1.2.2",
+    "eslint-plugin-promise": "^4.2.1",
+    "eslint-plugin-standard": "^5.0.0",
+    "esm": "^3.2.25",
+    "inquirer": "^8.1.1",
+    "mkdirp": "^0.5.1",
+    "mocha": "^8.4.0",
+    "moment": "^2.29.1",
```

Comment:
Just for advice: Is it better to use day.js instead of monent.js ? https://medium.datadriveninvestor.com/https-medium-com-sabesan96-why-you-should-choose-day-js-instead-of-moment-js-9cf7bb274bbd